README.md

```diff
@@ -3,61 +3,77 @@
 [![npm version](https://badge.fury.io/js/axios-curlirize.svg)](https://badge.fury.io/js/axios-curlirize)
 
 # Description
+
```

Comment:
Although I love the shoutout (😄 ) and the description, this description should be identical to the one on the main repo, in my opinion.